examples/README.txt

```diff
@@ -6,10 +6,9 @@ Examples
 End-to-end examples show how you can use PyFluent. If the PyFluent ``ansys-fluent-core`` package is installed on your machine,
 you can download these examples as Python files or Jupyter notebooks and run them locally.
 
-A collection of PyFluent examples can be found at `pyfluent-examples <https://github.com/ansys/pyfluent-examples>`_.
-
 .. toctree::
    :maxdepth: 1
    :hidden:
 
-.. 00-fluent/README.txt
\ No newline at end of file
+   00-fluent/README.txt
+   Miscellaneous Examples <https://examples.fluent.docs.pyansys.com/version/dev/examples/index.html>    
```

Comment:
```suggestion
   Miscellaneous Examples <https://examples.fluent.docs.pyansys.com/version/dev/index.html>    
```

Reply:
I think going straight to `https://examples.fluent.docs.pyansys.com/version/dev/**examples**/index.html` is better in this case

Reply:
@raph-luc, 
https://examples.fluent.docs.pyansys.com/version/dev/index.html
is referencing home, and it has “Back to PyFluent Stable Example” button. Let's commit this.

Reply:
With the recent changes from https://github.com/ansys/pyfluent-examples/commit/423862320331b33c364e29110be76f4df09eac83 (I think I checked before these were online, I didn't see them before) 

https://examples.fluent.docs.pyansys.com/version/dev/index.html is indeed better now. Thanks @abhishekchitwar @Revathyvenugopal162